elixir/apps/domain/lib/domain/gateways.ex

```diff
@@ -147,6 +158,13 @@ defmodule Domain.Gateways do
     {:ok, gateways}
   end
 
+  def list_gateways_for_resource(%Resources.Resource{} = resource) do
```

Comment:
@bmanifold is this function used? I can't find where. If it's used in front-end logic we need to have a subject here, otherwise we should just drop unused function.

Reply:
Yeah, let's drop this. This a function without clear use and tests.

Reply:
Sounds good.  Will do.

Reply:
This has been removed.